### DISCUSSION TITLE
/api/v1/status/ipsec seems to be gone in the V2 API?

### DISCUSSION THREAD

**Discussion by mauricevergeer** [jaredhendrickson13/pfsense-api#571] [Q&A]
Today I've installed the V2 API, but i'm trying to figure out a way to get the status data of my IPSec tunnels, in the V1 API there was this endpoint: /api/v1/status/ipsec, but in the V2 API there is no such endpoint that gives the status of my IPSec tunnel(s)?

**Comment by jaredhendrickson13** [comment]
That endpoint just hasn't been ported to v2 yet. The biggest difference between v1 and v2, is v2 requires us to fully define the schema of objects, whereas v1 just returns the output of the pfSense config/function directly (which makes it unpredictable and hard to document). It takes time to fully define the schema of some these pfSense functions because they're not super well documented, and a lot of time the schema is variable and can change due to many different factors. This one is getting pretty close however. I have schema is mostly mapped out, it just needs to be written out as Model classes the API can understand.

Work on v2.2.0 is a little ahead of schedule, it should be out by the end of the week or so. I'll put a little focus on this one and see if I can get into that release.

**Reply by mauricevergeer** [reply]
Hi Jared, 

Thanks so much for your reaction, please let me know if can test anything! #happytohelp
